### ISSUE TITLE
some wish to usability )

### ISSUE THREAD

**Issue by Kadiart** [eternicode/bootstrap-datepicker#1335]
hi there! 
i think range mode of datepicker (with 1 month min view) would be more useful and clear If value of end input would be equal to last day of month.

thanks

**Comment by Kadiart** [comment]
may be someone know, what to fix or add in scource code to get it?

**Comment by jnthnclrk** [comment]
I agree. I wonder if it's possible to force this behaviour in the meantime?

**Comment by enyosolutions** [comment]
+1 on that. Did any one manage to do it ?
